gcloud/test_credentials.py

```diff
@@ -176,11 +176,13 @@ def _get_pem_key(credentials):
         SIGNATURE_STRING = 'dummy_signature'
         with _Monkey(MUT, RSA=rsa, PKCS1_v1_5=pkcs_v1_5,
                      SHA256=sha256, _get_pem_key=_get_pem_key):
-            self.assertRaises(NameError, self._callFUT,
+            self.assertRaises(UnboundLocalError, self._callFUT,
```

Comment:
It doesn't. I noticed it at some point throughout and decided to fix. (`UnboundLocalError` is a subclass of `NameError`)